hicommonwealth/commonwealth
# `[Launchpad Discovery]:` Token Trade From Explore Page - Buy Mode

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8804

## Description of Changes
Added ability to buy tokens from the explore page. 

## "How We Fixed It"
N/A

## Test Plan
1. Create a new token
2. Visit the explore page and attempt to buy that token (ignore the pricing data - its placeholder)
3. In the buy modal verify you correctly see the eth amount in your selected address
4. Verify the dollar amount you enter correctly translates into eth amount
5. Verify you are able to see receipt details correctly (tokens bought are hardcoded for now)
6. Verify you are able to buy the token successfully
7. Verify the modal closes after the token is bought successfully
8. Verify you get joined to the token community if not already joined after a successful token purchase.

## Deployment Plan
N/A

## Other Considerations
N/A
